static/app/views/alerts/rules/issue/setupMessagingIntegrationButton.tsx

```diff
@@ -0,0 +1,109 @@
+import {useEffect} from 'react';
+import styled from '@emotion/styled';
+
+import {openModal} from 'sentry/actionCreators/modal';
+import {Button} from 'sentry/components/button';
+import {Tooltip} from 'sentry/components/tooltip';
+import {t} from 'sentry/locale';
+import PluginIcon from 'sentry/plugins/components/pluginIcon';
+import {space} from 'sentry/styles/space';
+import type {Organization} from 'sentry/types/organization';
+import type {Project} from 'sentry/types/project';
+import {trackAnalytics} from 'sentry/utils/analytics';
+import {useApiQuery} from 'sentry/utils/queryClient';
+import MessagingIntegrationModal from 'sentry/views/alerts/rules/issue/messagingIntegrationModal';
+
+type Props = {
+  organization: Organization;
+  projectSlug: string;
+  refetchConfigs: () => void;
+};
+
+function SetupMessagingIntegrationButton({
+  organization,
```

Comment:
Instead of passing in `organization`, you can use `useOrganization()` inside this component. Makes it so that you don't need to pass that prop around everywhere.

In your test, you can add your test org to the context by doing:

```tsx
render(<Component />, { organization: mockOrganization })
```